docs/user/faq/troubleshooting.md

```diff
@@ -2,106 +2,118 @@
-
-<p align="center">
-    <img src="../../../img/recovery_token.png"/>
-</p>
-
-You should save your token in a secure place like a password manager, paper, etc. In this case, we would save this:
-
-```
-4LMB9w3l50Yljwr6bIgQ
-```
-
-## Can I recover my recovery token?
-
-Firstly, we have to go to the login page of our DAppNode:
-
-<p align="center">
+<details>
+  <summary>Help! I can't access my DAppNode!</summary>
+  <div>
+    <div>
+    If it is your first time trying to access your DAppNode, please try one of the following steps:
+    <br /><br />
+<ul>
+    <li>If you are connected to the same network as your DappNode via wired Ethernet, try to access the UI using the address <a href="http://dappnode.local">http://dappnode.local</a></li>
+    <li>If you've bought a DAppNode or installed it on a machine with integrated Wi-Fi, try to access the Wi-Fi hotspot 'DAppNodeWifi'.</li>
+</ul>
+
+If all of the above does not work, visit our <a href="https://discord.com/invite/dappnode">Discord</a> and ask for help in the #support channel.
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>Why can't I connect to my DAppNode via VPN?</summary>
+  <div>
+    <div>
+    If you have set up OpenVPN and you cant access your DAppNode, remember that you might need to set up port forwarding on your router. Sometimes the feature called 'UPnP' enables these ports on your router automatically but this sometimes does not work or is not available on your router.
+    <br /><br />
+
+To do this, you need to access your router configuration and "port-forward" the following ports: TCP Port 8092 and UDP Port 1194. 
+
+In case you have set up Wireguard, you might have to create two profiles:
+
+<ul>
+    <li>A Local profile: This profile is used when you are connecting from within the same network that the DAppNode resides in.</li>
+    <li> A Remote profile: This profile is used when you are connecting from a different network than the one where your DAppNode resides in.</li>
+</ul>
+  When you create the VPN profiles, there is a link that generates the profile for either local usage or remote usage.
+  <br /><br />
+You can also check out our guides on how to configure VPN access <a href="/user/guides/access/vpn">here</a>.
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>Can I recover a lost admin password?</summary>
+  <div>
+    <div>
+    You can not outright recover your admin password but you can regain access to your DAppNode if you wrote down the recovery token provided during the inital setup. 
+    <br /><br />
+    More about the initial setup can be found <a href="/user/quick-start/first-steps">here</a>
+    <br /><br />
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>How do I recover a lost admin password using my recovery token?</summary>
+  <div>
+    <div>
+    Visit the login page of your DAppNode:
+    <br /><br />
+    <p align="center">
     <img src="../../../img/login.png"/>
```

Comment:
```suggestion
    <img src="../../../static/img/login.png"/>
```
this is why images arent rendering, the path is wrong on like all images and needs this change for each image, should be done in a single PR/ i think though since its throughout the docs

Reply:
Running docusaurus with yarn start locally shows no missing images with the current path (no static in path).

Reply:
https://docusaurus.io/docs/static-assets 

Says: `the image /static/img/docusaurus.png will be served at /img/docusaurus.png`